### PR TITLE
Update to docs around prometheus.yml configs

### PR DESCRIPTION
**Pull request by james-laing**
1. Added some `scrape_config` details which were missing.
2. Removed alerting and rules config to clean up file and maintain focus on simple prometheus config. Updated IP addressed with dummy fqdn.com
3. Updated dummy credentials which look legit with clear dummy data which can be easily differentiated between array1 and array2 configs.
4. Recommended to collect `/metrics/directories` metrics every 15 mins for large environments.

### CODE REVIEW

File: extra/prometheus/prometheus.yml [hunk 1]
```diff
@@ -1,28 +1,15 @@
 global:
   scrape_interval:     30s # Set the scrape interval to every 30 seconds. Default is every 1 minute.
-  scrape_timeout:      30s # Set the scrape output to every 30 seconds. Default is every 1 minute.
+  scrape_timeout:      30s # Set the scrape timeout to shorter or match. Default is every 1 minute.
```
Comment (sdodsley) [inline extra/prometheus/prometheus.yml:3]: when you say match, you should specifiy match `scrape_interval`

File: extra/prometheus/prometheus.yml [hunk 2]
```diff
@@ -122,14 +122,14 @@ scrape_configs:
...
       endpoint: ['arrayname02.fqdn.com']
 
     static_configs:
     # purefa openmetrics exporter
     - targets:
-      - 10.0.2.10:9490
+      - purefa-openmetrics-exporter.fqdn.com:9490
       labels:
         location: uk
         site: London
```
Comment (sdodsley) [inline extra/prometheus/prometheus.yml:135]: This should be fr and Paris

**Review by sdodsley — CHANGES_REQUESTED** [review]
Couple of tiny issues.

File: extra/grafana/README.md [hunk 3]
```diff
@@ -65,71 +65,75 @@ Dashboards may have limited functionality with earlier versions and some modific
...
+            # location, site and env are specific to your environment. Feel free to add more labels but maintain these three to minimize changes to Grafana which is expecting to use location, site and env as filter variables. 
+            location: uk
+            site: London
+            env: production
+
+      # Repeat for the above for end points:
+      # /metrics/volumes
+      # /metrics/hosts
+      # /metrics/pods
+      # /metrics/directories
```
Comment (sdodsley) [inline extra/grafana/README.md:115]: maybe add something here about 15 minutes scrape time for directories

**Review by sdodsley — CHANGES_REQUESTED** [review]

File: extra/prometheus/prometheus.yml (hunk 1 above)
Comment (james-laing) [reply extra/prometheus/prometheus.yml]: Resolved in commit de6ba1b4812834ef195edb505000770802cc245b

File: extra/prometheus/prometheus.yml (hunk 2 above)
Comment (james-laing) [reply extra/prometheus/prometheus.yml]: Well spotted! Resolved in commit de6ba1b4812834ef195edb505000770802cc245b

File: extra/grafana/README.md (hunk 3 above)
Comment (james-laing) [reply extra/grafana/README.md:115]: Resolved in commit [b3d29ee](https://github.com/PureStorage-OpenConnect/pure-fa-openmetrics-exporter/pull/87/commits/b3d29ee93cb226ac653ecbdbef90e65ed1c36196)

**Comment by james-laing** [comment]
Thanks for the feedback @sdodsley. All comments have been addressed and resolved.

File: extra/prometheus/prometheus.yml [hunk 4]
```diff
@@ -1,28 +1,15 @@
 global:
   scrape_interval:     30s # Set the scrape interval to every 30 seconds. Default is every 1 minute.
-  scrape_timeout:      30s # Set the scrape output to every 30 seconds. Default is every 1 minute.
+  scrape_timeout:      30s # Set the scrape timeout to shorter or match or match scrape_interval. Default is every 1 minute.
```
Comment (sdodsley) [inline extra/prometheus/prometheus.yml:3]: now this doesn't make sense. you have match twice

**Review by sdodsley — CHANGES_REQUESTED** [review]

File: extra/prometheus/prometheus.yml (hunk 4 above)
Comment (james-laing) [reply extra/prometheus/prometheus.yml]: Copy, paste and post edit typo.
Fixed in commit [a437f30](https://github.com/PureStorage-OpenConnect/pure-fa-openmetrics-exporter/pull/87/commits/a437f30e6a57c70102a98c6e37caacc903612a57)

**Comment by james-laing** [comment]
Amendments made to `extra/grafana/README.md` supporting documents to make consistent across documentation set.

**Review by sdodsley — APPROVED** [review]

**Review by chrroberts-pure — APPROVED** [review]

**Comment by chrroberts-pure** [comment]
LGTM.

File: extra/grafana/README.md [hunk 5]
```diff
@@ -65,71 +65,78 @@ Dashboards may have limited functionality with earlier versions and some modific
 1. Install Prometheus on your chosen OS platform ([prometheus-docs][2]).
 
 2. Generate an API token from your chosen user account or create a new readonly user.
```
[the remark this answers is not in the document]
Comment (sdodsley) [reply extra/grafana/README.md:68]: I agree that this would be good to add

[the remark this answers is not in the document]
Comment (james-laing) [reply extra/grafana/README.md:68]: Added to commit https://github.com/PureStorage-OpenConnect/pure-fa-openmetrics-exporter/pull/87/commits/9b6a2a7ec188557cbcb5d93b4f7bfc48b74f4691

**Review by sdodsley — APPROVED** [review]

**Pull request merged**